frontend/src/app/about/page.tsx

```diff
@@ -0,0 +1,73 @@
+"use client";
+
+import Header from "@cloudscape-design/components/header";
+import { COMMIT_RECENT_TAG, COMMIT_SHA, COMMIT_DATE } from "@/lib/env";
+import {
+  Box,
+  Container,
+  Icon,
+  KeyValuePairs,
+  Link,
+  SpaceBetween,
+} from "@cloudscape-design/components";
+
+export default function Page() {
+  return (
+    <SpaceBetween size="l">
+      <Header variant="h1">About Migration Assistant</Header>
+      <Container>
+        <SpaceBetween size="l">
+          <KeyValuePairs
+            columns={3}
+            items={[
+              {
+                label: "Version",
+                value: COMMIT_RECENT_TAG,
+                info: (
+                  <Link
+                    external
+                    variant="info"
+                    href={`https://github.com/opensearch-project/opensearch-migrations/releases/tag/${COMMIT_RECENT_TAG}`}
+                  ></Link>
+                ),
+              },
+              {
+                label: "Commit SHA",
+                value: COMMIT_SHA,
+                info: (
+                  <Link
+                    external
+                    variant="info"
+                    href={`https://github.com/opensearch-project/opensearch-migrations/commit/${COMMIT_SHA}`}
```

Comment:
This seemed to produce a link to a commit that Github could not find: https://github.com/opensearch-project/opensearch-migrations/commit/0be5c448e

Reply:
Being unable to find the commit is an issue for build done outside this repo - it will happen for builds from forks.  I consider this a feature to lets us trace exactly the build a customer is running if it is non-standard.